src/ts/components/subtitleoverlay.ts

```diff
@@ -85,6 +85,13 @@ export class SubtitleOverlay extends Container<SubtitleOverlayConfig> {
 
       this.applyConfToDom(labelToAdd.getDomElement());
 
+      if (event.position != null) {
+        let domElement = labelToAdd.getDomElement();
+        domElement.css('position', 'fixed');
+        domElement.css('left', `${event.position.column}em`);
```

Comment:
The subtitle style should be able dynamically adapt (e.g. when switching the subtitle track from CEA-608 to something else), so detecting CEA-608 and adjusting the style when a caption arrives is fine.